.github/workflows/publish-dev.yml

```
@@ -0,0 +1,98 @@
+name: Publish Dev Packages
+
+on:
+  #push:
+  #  branches:
+  #    - 'epic-v[0-9]+.[0-9]+.x'
+  workflow_dispatch:
+    inputs:
+      version_suffix:
+        description:
+          'Version and tag suffix (used in tag: dev-{suffix} and version:
+          x.y.z-dev.{suffix}.commit)'
+        required: true
+        type: string
+      reason:
+        description: 'Reason for manual publish'
+        required: false
+        default: 'Manual trigger by maintainer'
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  publish-dev:
+    name: Publish development versions to npm
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v5
+        with:
+          fetch-depth: 0
```

nitpick, but this should be enough, shouldn't it?

```suggestion
          fetch-depth: 1
```